components/fxa-client/ffi/src/lib.rs

```diff
@@ -133,6 +133,44 @@ pub extern "C" fn fxa_get_connection_success_url(
     })
 }
 
+/// Get the url to open the user's account-management page.
+///
+/// # Safety
+///
+/// A destructor [fxa_str_free] is provided for releasing the memory for this
+/// pointer type.
+#[no_mangle]
+pub extern "C" fn fxa_get_manage_account_url(
+    handle: u64,
+    entrypoint: FfiStr<'_>,
+    error: &mut ExternError,
+) -> *mut c_char {
+    log::debug!("fxa_get_manage_account_url");
+    ACCOUNTS.call_with_result_mut(error, handle, |fxa| {
+        fxa.get_manage_account_url(entrypoint.as_str())
+            .map(|u| u.to_string())
```

Comment:
nit: prefer into_string to to_string for cases like this.

Also, I suspect a fully updated clippy will want this to be `.map(Url::into_string)`

Ditto below.